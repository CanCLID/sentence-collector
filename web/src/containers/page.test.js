import React from 'react';
import { render } from '@testing-library/react';

import Page from './page';
import Header from '../components/header';
import HeaderBtn from '../components/header-btn';

jest.mock('../components/header');
jest.mock('../components/header-btn');

test('should render page', () => {
  Header.mockReturnValue('...Header...');
  HeaderBtn.mockReturnValue('...HeaderBtn/...');
  const { container } = render(<Page/>);
  expect(container).toMatchSnapshot();
});
