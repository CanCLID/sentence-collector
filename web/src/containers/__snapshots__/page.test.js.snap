// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render page 1`] = `
<div>
  ...Header...
  <div
    id="page"
  >
    ...HeaderBtn/...
    <main />
  </div>
</div>
`;
